### PR TITLE
J170: + type for episode

### PR DESCRIPTION
**Pull request by EugeniusGR**
image

### CODE REVIEW

File: frontend/src/components/episode/episode.tsx [hunk 1]
```diff
@@ -115,7 +115,8 @@ const Episode: React.FC = () => {
                 )}
                 <h1 className={styles.title}>{episode.name}</h1>
                 <p className={styles.description}>{episode.description}</p>
-                <p className={styles.status}>Status: {episode.status}</p>
+                <p className={styles.episodeInfo}>Type: {episode.type}</p>
+                <p className={styles.episodeInfo}>Status: {episode.status}</p>
```
Comment (what1s1ove) [inline frontend/src/components/episode/episode.tsx]:
```suggestion
<dl>
    <dt>Type: </dt>
    <dd>{episode.type}</dd>
    <dt>Status:</dt>
    <dd>{episode.status}</dd>
</dl>
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/episode/styles.module.scss [hunk 2]
```diff
@@ -111,6 +111,11 @@ h3 {
   margin: 0;
 }
 
+dt {
+  float: left;
+  margin-right: 5px;
+}
```
Comment (what1s1ove) [inline frontend/src/components/episode/styles.module.scss]:
```suggestion
```

File: frontend/src/components/episode/styles.module.scss [hunk 3]
```diff
@@ -95,9 +95,9 @@
   width: 450px;
 }
 
-.status {
-  font-size: 12px;
-  margin: 15px 0;
+.episodeInfo {
+  font-size: 14px;
+  margin: 5px 0;
 }
```
Comment (what1s1ove) [inline frontend/src/components/episode/styles.module.scss]:
```suggestion
```
you don't use it, right?

File: frontend/src/components/episode/episode.tsx [hunk 4]
```diff
@@ -115,7 +115,12 @@ const Episode: React.FC = () => {
                 )}
                 <h1 className={styles.title}>{episode.name}</h1>
                 <p className={styles.description}>{episode.description}</p>
-                <p className={styles.status}>Status: {episode.status}</p>
+                <dl>
+                  <dt>Type: </dt>
+                  <dd>{episode.type}</dd>
+                  <dt>Status:</dt>
+                  <dd>{episode.status}</dd>
+                </dl>
```
Comment (what1s1ove) [inline frontend/src/components/episode/episode.tsx]:
```suggestion
                <dl>
                   <div>
                                     <dt>Type: </dt>
                  <dd>{episode.type}</dd>
                   </div>
                   <div>
                                     <dt>Status:</dt>
                  <dd>{episode.status}</dd>
                   </div>
                </dl>
```
add style by classes

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/episode/styles.module.scss [hunk 5]
```diff
@@ -96,8 +96,16 @@
...
 
 .episodeInfo {
-  font-size: 14px;
   margin: 5px 0;
+  font-size: 12px;
+
+  .episodeInfoTitle {
+    float: left;
+  }
```
Comment (what1s1ove) [inline frontend/src/components/episode/styles.module.scss]: we don't use the float for layout
use `display: flex`;

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Comment by EugeniusGR** [comment]
image

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**